### PR TITLE
Add support for `XR_FB_display_refresh_rate`

### PR DESCRIPTION
**Pull request by Timbals**

### CODE REVIEW

File: openxr/src/display_refresh_rate.rs [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        let ext = self
+            .inner
+            .instance
+            .exts()
+            .fb_display_refresh_rate
+            .as_ref()
+            .expect("XR_FB_display_refresh_rate not loaded");
+        unsafe {
+            let mut out = MaybeUninit::uninit();
+            (ext.get_display_refresh_rate)(self.as_raw(), out.as_mut_ptr());
```
Comment (Ralith) [inline openxr/src/display_refresh_rate.rs:46]:
```suggestion
            cvt((ext.get_display_refresh_rate)(self.as_raw(), out.as_mut_ptr()))?;
```

File: openxr/src/display_refresh_rate.rs [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    pub fn request_display_refresh_rate(&self, display_refresh_rate: f32) -> Result<()> {
+        let ext = self
+            .inner
+            .instance
+            .exts()
+            .fb_display_refresh_rate
+            .as_ref()
+            .expect("XR_FB_display_refresh_rate not loaded");
+        cvt(unsafe { (ext.request_display_refresh_rate)(self.as_raw(), display_refresh_rate) })?;
+        Ok(())
```
Comment (Ralith) [inline openxr/src/display_refresh_rate.rs:65]: nit:
```suggestion
        cvt(unsafe { (ext.request_display_refresh_rate)(self.as_raw(), display_refresh_rate) })
```

**Review by Ralith — CHANGES_REQUESTED** [review]
Thanks!

File: openxr/src/display_refresh_rate.rs (hunk 2 above)
Comment (Timbals) [reply openxr/src/display_refresh_rate.rs]: This doesn't work because of the return type being `Result<()>` and not `Result<sys::Result>`.

**Review by Ralith — APPROVED** [review]

**Pull request merged**